app/assets/stylesheets/common/base/login-signup-page.scss

```diff
@@ -42,6 +42,9 @@ body.activate-account-page {
 body.login-page,
 body.signup-page {
   background-color: var(--secondary);
+  &.redirecting {
+    display: none;
+  }
```

Comment:
Not a huge fan of using `display:none` here. 

For now, to keep this PR super simple, let's render the page in full and have it redirect. It'll make the template easier to reason about. The behaviour would be the same as on the modal, so no regression really. 

Later we can make this smarter, with a loading icon while the model is loading or something similar.